app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+    alive: list["Animal"] = []
```

Comment:
The `alive` list is a class variable, which means it is shared across all instances of `Animal`. If you want each instance to have its own `alive` list, consider making it an instance variable instead.